nncf/quantization/algorithms/fast_bias_correction/backend.py

```diff
@@ -121,7 +114,7 @@ def create_input_data(
 
     @staticmethod
     @abstractmethod
-    def get_bias_value(node: NNCFNode, nncf_graph: NNCFGraph, model: TModel) -> np.ndarray:
+    def get_bias_value(node: NNCFNode, nncf_graph: NNCFGraph, model: TModel) -> NNCFTensor:
```

Comment:
"Backend entities" etc. are constrained to return `NNCFTensors` whenever the corresponding methods are supposed to be used from common code